BCP/Session Lifecycle Management.md

```diff
@@ -0,0 +1,147 @@
+---
+title: JSON-RPC Provider Session Lifecycle Management with CAIP-25 Sessions BCP
+author: [Alex Donesky] (@adonesky1)
+discussions-to: TBD
+status: Draft
+type: Standard
+created: 2024-06-07
+requires: 25, 217, 285, 307, 308
+---
+
+## Simple Summary
+
+This backwards-compatible extension of the [CAIP-25][] standard defines new JSON-RPC methods for managing the lifecycle of authorizations within a session.
+These methods allow dapps and wallets to dynamically adjust authorizations, providing more granular control and better user experience.
+Additionally, it allows for session management without mandatory [sessionIds][CAIP-171], offering more flexibility in handling sessions in single-session contexts.
+
+## Abstract
+
+This proposal aims to extend the CAIP-25 standard by defining new JSON-RPC methods for managing the lifecycle of authorizations within a session. These methods allow dapps and wallets to dynamically adjust authorizations, providing more granular control and better user experience. Additionally, it allows for session management without mandatory sessionIds, offering more flexibility in handling sessions.
```

Comment:
By loosening the earlier requirement that both caller and wallet maintain a session identifier to track the session across potentially shared or insecure transports, single-session transport options can take advantage of direct RPC calls to do more explicit session-management.
The equivalence across different transports can be counter-intuitive, so the following diagrams and examples are provided for apples-to-apples comparisons.